lib/image.js

```diff
@@ -60,8 +62,8 @@ async function getDataUri(file, options) {
   return `data:${file.mime};base64,${Buffer.from(file.contents, 'binary').toString('base64')}`;
 }
 
-async function resolve(filepath, {assetPaths, maxFileSize, resolver = () => undefined}) {
-  const filter = (resource) => assertSize(resource, maxFileSize);
+async function resolve(filepath, {assetPaths, maxFileSize, largeFileCallback, resolver = () => undefined}) {
+  const filter = (resource) => assertSize(resource, maxFileSize, !largeFileCallback);
```

Comment:
This could also be:

```js
  const filter = (resource) => largeFileCallback ? resource : assertSize(resource)
```

Then you don't have to touch the other function.

Reply:
Thanks for the tip. Unfortunately will give you a callback on all files, not just the large ones.